protractor.conf.js

```diff
@@ -1,123 +1,53 @@
-﻿var paths = require('./paths.config.js');
+﻿'use strict';
+
+var paths = require('./paths.config.js');
```

Comment:
Er ESLint glad?

Reply:
Der er en jasmine del under onPrepare som eslint ikke er glad for. Men denne er protractor tilgengæld meget glad for.